### PR TITLE
📝 Readme enhancement

### PR DESCRIPTION
**Pull request by ppratikcr7**
This separate PR contains updated readme with dynamic badges and other details related to `Upgrade` to help reach more people. Please review and discuss if we need edit/add/remove anything.

### CODE REVIEW

**Comment by danoswaltCL** [comment]
@amurphy-cl can you give us your thoughts on the new documentation? we're assigning you as code-reviewer for this one

File: README.md [hunk 1]
```diff
@@ -1,8 +1,103 @@
...
+
+Navigate into root of project and run these commands to install dependencies:
+
+$ cd Upgrade
+$ make setup-local
+```
+
+```
+NOTE: If this make command says nix not found and you are using zsh on mac, it may need to be manually added to PATH. OR, just try running it in bash
+```
```
Comment (bcb37) [inline README.md:63]: This NOTE text appears in a horizontal scroll, rather than wrapping. I think it probably doesn't need to be inside the ```'s.

**Review by bcb37 — CHANGES_REQUESTED** [review]
Just one nit. Otherwise it looks good, just need to resolve the merge conflicts.

**Comment by ppratikcr7** [comment]
@bcb37 Now the PR is ready for approval and merging.

**Review by bcb37 — APPROVED** [review]

**Pull request merged**